scripts/linux/buildBranch.sh

```diff
@@ -225,11 +225,6 @@ build_solution()
         RES=1
     fi
 
-    echo "Building IoT Edge Samples solution"
-    $DOTNET_ROOT_PATH/dotnet build \
-        -c $CONFIGURATION \
-        -o "$BUILD_BINARIESDIRECTORY" \
-        "$ROOT_FOLDER/samples/dotnet/Microsoft.Azure.Devices.Edge.Samples.sln"
```

Comment:
This is no longer a separate solution. All the csproj files are included in the main sln.